src/app/components/chips/chips.spec.ts

```diff
@@ -96,14 +96,14 @@ describe('Chips', () => {
         let data;
         chips.onAdd.subscribe((value) => (data = value));
         const addItemSpy = spyOn(chips, 'addItem').and.callThrough();
-        const onKeydownSpy = spyOn(chips, 'onKeydown').and.callThrough();
+        const onKeydownSpy = spyOn(chips, 'onKeyDown').and.callThrough();
```

Comment:
the variable name should be `onKeyDownSpy` (uppercase D) maybe for consistency and the same with the others, IMO.
What do you think @ashikjs?